src/redux/books/books.js

```diff
@@ -1,45 +1,67 @@
-const ADD_BOOK = 'ADD_BOOK';
-const REMOVE_BOOK = 'REMOVE_BOOK';
-
-const initialState = [
-  {
-    id: Math.floor(Math.random() * 100),
-    author: 'JK Rowlings',
-    title: 'Harry Potter',
-  },
-  {
-    id: Math.floor(Math.random() * 100),
-    author: 'Ken Follet',
-    title: 'Fire',
-  },
-  {
-    id: Math.floor(Math.random() * 100),
-    author: 'Kalani Pickhart',
-    title: 'I will die in a foreign land',
-  },
-];
-
-const bookReducer = (state = initialState, action) => {
-  switch (action.type) {
-    case ADD_BOOK: { return [...state, action.book]; }
-    case REMOVE_BOOK: return state.filter((book) => book.id !== action.id);
-    default: return state;
+import axios from '../axiosInstance';
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements. Kindly ensure that you use `createAsyncThunk` to create thunk action creators. While using that, you will need to 
`import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';`
and also import axios just as you have done.

Here is a typical example:
```
import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
import axios from 'axios'; 


export const getBooks = createAsyncThunk(
  'books/getBooks',
  async () => {
    const response = await axios.get(
      'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/',
    );
    return response.data;
  },
); 
```